src/cita.js

```diff
@@ -0,0 +1,17 @@
+const {
+  default: AppChain
+} = require('@appchain/base')
```

Comment:
[@crytape/cita-sdk](https://www.npmjs.com/package/@cryptape/cita-sdk) is available

Reply:
ok